docs/installation.md

```diff
@@ -281,9 +285,12 @@ with a slug of `cfgov`.
 If you're installing this fresh, the initial data you receive will not be
 as extensive as you'd probably like it to be.
 
-You can get a database dump using the `cf.gov-dump-rdbms`
-Jenkins job. Download the `sql.gz` file,
-unzip it, and then run:
+You can get a database dump by:
+
+1. Going to https://github.cfpb.gov/CFGOV/platform/wiki/Database-downloads
```

Comment:
@JeffreyMFarley can you please redact this URL? Policy is that it should remain internal-only, and we typically refer to it here as `[GHE]`. If you amend your commit and force push you can remove it from the public history.